doc/manual/src/language/operators.md

```diff
@@ -63,6 +63,8 @@ The result is a [Boolean] value.
 
 [Has attribute]: #has-attribute
 
+Complexity is O(log₂(*n*)) for *n* attributes in the *attrset*
```

Comment:
```suggestion
After evaluating *attrset* and *attrpath*, the computational complexity is O(log(*n*)) for *n* attributes in the *attrset*
```

- exclude evaluation of the attrset, which is arbitrary
- remove base 2, because log bases are constant factors, which don't make for different asymptotic classes

Btw looks like the syntax example is incomplete. Attrpath can be a computed value: `a ? ${foo bar}`